format-parquet/src/main/java/io/cdap/plugin/format/parquet/input/ParquetInputFormatProvider.java

```diff
@@ -80,7 +81,9 @@ public Schema detectSchema(FormatContext context, InputFiles inputFiles) throws
       }
       try (ParquetFileReader parquetFileReader = ParquetFileReader.open(new HadoopInputFile(inputFile), readOptions)) {
         MessageType parquetSchema = parquetFileReader.getFooter().getFileMetaData().getSchema();
-        return new AvroToStructuredTransformer().convertSchema(new AvroSchemaConverter().convert(parquetSchema));
+        Schema schema = new AvroToStructuredTransformer().convertSchema(
+                new AvroSchemaConverter().convert(parquetSchema));
```

Comment:
fix indentation

Reply:
This is not resolved, the indentation is incorrect.

Reply:
Fixed indentation

Reply:
this has not been fixed